app/controllers/topics_controller.rb

```diff
@@ -150,7 +150,7 @@ def destroy
     @topic = Topic.find(params[:id])
     authorize @topic
 
-    @topic.delete
+    @topic.destroy
```

Comment:
@devvmh this is interesting: I discovered that by using `delete` here, our `dependent: :destroy` methods weren't taking effect and deleting mappings, so this is actually a bug on prod... we've got orphaned data happening